### PR TITLE
Feature/34 home page

### LINKED ISSUE #34: Create Home + menu pages

### PR DESCRIPTION
**Pull request by MetalBrackets**
close #34

### CODE REVIEW

File: assets/assets/main.css [hunk 1]
```diff
@@ -4,36 +4,42 @@
...
-        display: flex;
-        place-items: center;
-    }
-
-    #app {
-        display: grid;
-        grid-template-columns: 1fr 1fr;
-        padding: 0 2rem;
+        background-color: hsla(66, 100%, 55%, 0.6);
+        border-radius: 100px;
```
Comment (MisterPeModder) [inline assets/assets/main.css:43]: You should *not* use global CSS, only Tailwind, and very rarely scoped CSS *inside* the components themselves.

File: assets/components/IconApp.vue [hunk 2]
```diff
@@ -0,0 +1,48 @@
+<template>
+    <div>
+        <button :class="['icons', 'cursor-pointer']" :style="{ backgroundColor: color }" @click="handleClick">
+            <font-awesome-icon :icon="iconClass" size="2x" />
+        </button>
+    </div>
+</template>
+
+<script setup lang="ts">
+const props = defineProps({
```
Comment (MisterPeModder) [inline assets/components/IconApp.vue:10]: Please use the typed variant of defineProps (doc at https://vuejs.org/guide/typescript/composition-api.html#typing-component-props)

File: assets/components/IconApp.vue [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    background-color: var(--color);
+    width: 72px;
+    height: 72px;
+    padding: 8px 8px;
+    font-size: 25px;
+    border-radius: 4px;
+    font-weight: 700;
+    border: 4px solid #4c324d;
+    box-shadow: 8px 8px #4c324d;
+    border-radius: 4px;
```
Comment (MisterPeModder) [inline assets/components/IconApp.vue:40]: Please use multiples of 0.25em for sizes (and their corresponding classes like p-1, p-2, m-4, etc...)

File: assets/components/NavBar.vue [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+import { RouterLink } from 'vue-router';
+import IconApp from './IconApp.vue';
+
+defineComponent({
+    name: 'NavBar',
+    components: {
+        RouterLink,
+        IconApp,
+    },
+});
```
Comment (MisterPeModder) [inline assets/components/NavBar.vue:12]: This function call is redundant, you safely remove it.

File: tsconfig.vitest.json [hunk 5]
```diff
@@ -2,6 +2,7 @@
   "extends": "./tsconfig.app.json",
   "exclude": [],
   "compilerOptions": {
+    "allowJs": true,
```
Comment (MisterPeModder) [inline tsconfig.vitest.json:5]: Do not use this, we are exclusively using TypeScript.

**Review by MisterPeModder — CHANGES_REQUESTED** [review]

**Review by angemarchand — APPROVED** [review]

**Pull request merged**